commands.py

```diff
@@ -27,17 +28,53 @@ def run(self, edit: sublime.Edit) -> None:
 
         region = completion.region
         display_text = completion.display_text or ""
+        completion_uuid = completion.uuid or ""
 
         if not region:
             return
 
         completion.hide()
         self.view.insert(edit, region[1], display_text)
 
+        session = self.session_by_name(self.session_name)
```

Comment:
The logic here is almost identical(except the Request name part), so we could extract it to a private method `_report_telemetry` and call it in both commands.